### PR TITLE
Origin/site updates

### PR DESCRIPTION
**Pull request by justinmclean**
some minor update to the site to fix a few broken things and correct some english

### CODE REVIEW

File: site/src/site/asciidoc/developers/team.adoc [hunk 1]
```diff
@@ -32,3 +32,12 @@ Sorted by first name:
 
  Frankfurt |Likes to swim outside the mainstream. He is really passionate about walking new paths. His first email to an Apache list on file was to the Cocoon project back in the year 2000. His direct involvement started in 2012 when he was elected to become an Apache Flex committer. Since then he became involved with a lot of Apache-internal groups and committees and was elected to become a Member of the Apache Software Foundation in 2015 and recently has taken up the role of mentoring several IoT podlings in the Apache Incubator. | a|image::team/cdutz.png[cdutz, 200, 200]
 
+| *Justin Mclean*
+
+Trainer and Consultant
+
+Various
+
+Justin Mclean been teaching for 10+ years and now splits his time between that and software development. He runs his own consulting company Class Software for 25+ years and has spoken at numerous conferences in his home of Australia and overseas. He's heavily involved in open source hardware and software, and runs the IoT meetup in Sydney. He currently teaches at an online collage and a university.
```
Comment (lfrancke) [inline site/src/site/asciidoc/developers/team.adoc]:
```suggestion
Justin Mclean has been teaching for 10+ years and now splits his time between that and software development. He runs his own consulting company Class Software for 25+ years and has spoken at numerous conferences in his home of Australia and overseas. He's heavily involved in open source hardware and software, and runs the IoT meetup in Sydney. He currently teaches at an online college and a university.
```

File: site/src/site/asciidoc/developers/website.adoc [hunk 2]
```diff
@@ -34,17 +34,17 @@ The site content itself is generated from `asciidoc` files (ending `.adoc`) whic
 
 Beyond the basic goodies, the build is also configured to generate images from ASCII data using the `asciidoctor-diagram` plugin.
 
-This allows us to generate images like the ones on the http://plc4x.apache.org/protocols/s7/index.html[S7 Protocol Description page]
+This allows us to generate images like the ones on the http://4x.apache.org/protocols/s7/index.html[S7 Protocol Description page]
```
Comment (lfrancke) [inline site/src/site/asciidoc/developers/website.adoc]: This link is broken

File: site/src/site/asciidoc/developers/website.adoc [hunk 3]
```diff
@@ -144,16 +144,16 @@ The `menu ref` items hereby reference standard menus provided by the `Maven` bui
...
 
-The build system needs to check-in content to the `asf-site` branch and usually ASF Jenkins nodes don't have the permissions to do that.
+The build system needs to check-in content to the `asf-site` branch and usually the ASF Jenkins nodes don't have the permissions to do that.
 
 In order to be able to push to the `asf-site` GIT branch, a dedicated build job is configured to build on nodes with the Jenkins label `git-websites`.
 
 Only on these machines are jobs allowed to push changes to a Git repo and here only to a branch named `asf-site`.
 
-See https://builds.apache.org/view/M-R/job/PLC4X/job/PLC4X%20Site/ for details on the PLC4X Jenkins Website build job.
+See https://builds.apache.org/view/M-R/job/4X/job/PLC4X%20Site/ for details on the PLC4X Jenkins Website build job.
```
Comment (lfrancke) [inline site/src/site/asciidoc/developers/website.adoc]: broken link

File: tools/maven-revealjs-asciidoctor-templte/src/main/asciidoc/index.adoc [hunk 4]
```diff
@@ -18,12 +18,12 @@
 ////
 
 include::_settings.adoc[]
-:presenter_name: Some Cool Guy
+:presenter_name: Some Cool Person
```
Comment (lfrancke) [inline tools/maven-revealjs-asciidoctor-templte/src/main/asciidoc/index.adoc]: I'm a bit confused by this and other changes. Haven't we merged this before? Those are good changes, so not opposing but it feels like I've seen it before...

File: site/src/site/asciidoc/developers/website.adoc (hunk 2 above)
Comment (justinmclean) [reply site/src/site/asciidoc/developers/website.adoc]: Link is working for me suggested link is not

File: site/src/site/asciidoc/developers/website.adoc (hunk 3 above)
Comment (justinmclean) [reply site/src/site/asciidoc/developers/website.adoc]: same here link is working changed link is not

File: tools/maven-revealjs-asciidoctor-templte/src/main/asciidoc/index.adoc (hunk 4 above)
Comment (justinmclean) [reply tools/maven-revealjs-asciidoctor-templte/src/main/asciidoc/index.adoc]: Thanks to git hub, It just added these changes to this branch, even though I created a new branch.

File: site/src/site/asciidoc/developers/website.adoc (hunk 2 above)
Comment (justinmclean) [reply site/src/site/asciidoc/developers/website.adoc]: We should change these to training links but we cont have any content that does that

**Pull request merged**